.github/workflows/continuous-delivery.yml

```diff
@@ -41,3 +41,65 @@ jobs:
         run: |
           tox -e syft.publish
           twine upload packages/syft/dist/*
+
+      - name: Set Grid package version
+        run: echo "GRID_VERSION=$(python packages/grid/VERSION)" >> $GITHUB_ENV
+      - name: Build and push `grid-backend` image to DockerHub
+        working-directory: ./packages
+        env:
+          IMAGE_URI: openmined/grid-backend
+        run: |
+          export DOCKER_URI=$IMAGE_URI:$GRID_VERSION
+          export DOCKER_URI_LATEST=$IMAGE_URI:latest
+          eval $(echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u ${{ secrets.DOCKER_LOGIN }} --password-stdin)
+          docker build \
+              -t $DOCKER_URI \
+              -t $DOCKER_URI_LATEST \
+              -f grid/backend/backend.dockerfile .
+          docker push $DOCKER_URI
+          docker push $DOCKER_URI_LATEST
+
+      - name: Build and push `grid-frontend` image to DockerHub
+        working-directory: ./packages/grid/frontend
```

Comment:
We set the working directory for each individual image, as Dockerfile for a few services is written to be run from their current path.